extra-x11/nvidia-prime/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=nvidia-prime
+PKGSEC=x11
+PKGDES="NVIDIA Prime Render Offload configuration and utilities"
```

Comment:
Considerations:

- Is "NVIDIA Prime Render Offload" a proper noun? AFAIK, only NVIDIA and "Prime" are, if I'm not mistaken, please do not capitalise the others.
- Is it "Prime," or "PRIME"?

Reply:
- Modified to "NVIDIA PRIME Render Offloading configuration and utilities".
- "PRIME" according to Arch Wiki.
This description was originally from the package https://www.archlinux.org/packages/extra/x86_64/nvidia-utils/